src/selectbox_abstract.coffee

```diff
@@ -108,6 +108,8 @@ class SelectBoxAbstract
         # Replicate changes from the original select box to the fake one
         @orig.on 'reform.sync change DOMSubtreeModified', @refresh
         
+        @body.on 'reform.close', => @close()
```

Comment:
I I do it on orig I would have to iterate trough every reformed and trigger it..